### ISSUE TITLE
Reduced serum Insulin-like growth factor binding protein 3 concentration

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11373]
**Preferred term label:**
IGFBP3

**Synonyms**
Reduced serum IGFBP-3 level
Reduced IGF binding protein-3 level

**Definition (free text, please give PubMed ID)**
PMID: 33122102 For subjects carrying AR STAT5B mutations, basal GH levels were  normal (ranging from 0.1 to 17.6 ng/mL), and when stimulated, GH 
concentrations were often elevated (4.0–53.8 ng/mL). **Serum** IGF-I,  I**GFBP-3** and ALS **concentrations in all cases were abnormally low**, and 
remained low after GH treatment, during IGF-I generation tests or during GH therapy.

PMID: 21396575 Basal GH levels were normal, and when stimulated, GH concentrations were frequently elevated. In one case (Case 1), GH reached a stimulated peak of 53.8 ng/ml, from a basal GH of 9.4 ng/ml. **Serum IGF-I, IGFBP-3 and ALS concentrations in all cases were abnormally low**
(Table 1), and remained low after GH treatment in an IGF-I generation test or during GH therapy.

**Parent term (use hpo.jax.org/app)**
Abnormal serum Insulin-like growth factor binding protein 3 level (see issue #11371)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:633 Laron syndrome
ORPHA:220465 Laron syndrome with immunodeficiency

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
see here for Reduced term:
https://github.com/obophenotype/human-phenotype-ontology/issues/11371
